Modules/Sources/WordPressKit/RemotePostParameters.swift

```diff
@@ -21,6 +21,7 @@ public struct RemotePostCreateParameters: Equatable {
     public var format: String?
     public var isSticky = false
     public var tags: [String] = []
+    public var otherTerms: [String: [String]] = [:]
```

Comment:
@kean I need to add similar code, like updating the new `otherTerms` variable, assigning its value to another instance, etc. I don't think I missed any places. But can you pay extra attention to reviewing this part? Thanks!